FWCore/Framework/src/EventProcessor.cc

```diff
@@ -436,47 +435,45 @@ namespace edm {
       }
     }
     /*
-    bool nRunsSet = false;
-     */
+      bool nRunsSet = false;
+    */
     unsigned int nConcurrentRuns =1;
     /*
-    if(nRunsSet = optionsPset.existsAs<unsigned int>("numberOfConcurrentRuns",false)) {
-    nConcurrentRuns = optionsPset.getUntrackedParameter<unsigned int>("numberOfConcurrentRuns");
-    }
-     */
+      if(nRunsSet = optionsPset.existsAs<unsigned int>("numberOfConcurrentRuns",false)) {
+      nConcurrentRuns = optionsPset.getUntrackedParameter<unsigned int>("numberOfConcurrentRuns");
+      }
+    */
     unsigned int nConcurrentLumis =1;
     /*
-    if(optionsPset.existsAs<unsigned int>("numberOfConcurrentLuminosityBlocks",false)) {
-    nConcurrentLumis = optionsPset.getUntrackedParameter<unsigned int>("numberOfConcurrentLuminosityBlocks");
-    } else {
+      if(optionsPset.existsAs<unsigned int>("numberOfConcurrentLuminosityBlocks",false)) {
+      nConcurrentLumis = optionsPset.getUntrackedParameter<unsigned int>("numberOfConcurrentLuminosityBlocks");
+      } else {
       nConcurrentLumis = nConcurrentRuns;
-    }
-     */
+      }
+    */
     //Check that relationships between threading parameters makes sense
     /*
-    if(nThreads<nStreams) {
+      if(nThreads<nStreams) {
       //bad
-    }
-    if(nConcurrentRuns>nStreams) {
+      }
+      if(nConcurrentRuns>nStreams) {
       //bad
-    }
-    if(nConcurrentRuns>nConcurrentLumis) {
+      }
+      if(nConcurrentRuns>nConcurrentLumis) {
       //bad
-    }
-     */
+      }
+    */
     //forking
     ParameterSet const& forking = optionsPset.getUntrackedParameterSet("multiProcesses", ParameterSet());
     numberOfForkedChildren_ = forking.getUntrackedParameter<int>("maxChildProcesses", 0);
     numberOfSequentialEventsPerChild_ = forking.getUntrackedParameter<unsigned int>("maxSequentialEventsPerChild", 1);
     setCpuAffinity_ = forking.getUntrackedParameter<bool>("setCpuAffinity", false);
     continueAfterChildFailure_ = forking.getUntrackedParameter<bool>("continueAfterChildFailure",false);
     std::vector<ParameterSet> const& excluded = forking.getUntrackedParameterSetVector("eventSetupDataToExcludeFromPrefetching", std::vector<ParameterSet>());
-    for(std::vector<ParameterSet>::const_iterator itPS = excluded.begin(), itPSEnd = excluded.end();
-        itPS != itPSEnd;
-        ++itPS) {
-      eventSetupDataToExcludeFromPrefetching_[itPS->getUntrackedParameter<std::string>("record")].insert(
-                                                std::make_pair(itPS->getUntrackedParameter<std::string>("type", "*"),
-                                                               itPS->getUntrackedParameter<std::string>("label", "")));
+    for(auto const& ps : excluded) {
+      eventSetupDataToExcludeFromPrefetching_[ps.getUntrackedParameter<std::string>("record")].insert(
```

Comment:
Yep.  I limited most of my changes to `subProcesses_`, but this one could easily go in, too.